openedx_learning/apps/authoring/publishing/api.py

```diff
@@ -81,6 +86,7 @@
     "contains_unpublished_changes",
```

Comment:
Do you think we can change the implementation of `contains_unpublished_changes` to use this new data, as an optimization? Or no? (not a blocker at all)